Administrator Guide/Performance Testing.md

```diff
@@ -162,18 +162,20 @@ within that host, and iozone-pathname is the full pathname of the iozone
 executable to use on that host. Be sure that every target host can
 resolve the hostname of host where the iozone command was run. All
 target hosts must permit password-less ssh access from the host running
-the command. For example:
+the command. 
+
+For example:
 
         export RSH=ssh
         iozone -+m ioz.cfg -+h my-ip-address -w -c -e -i 0 -+n -C -r 64k -s 1g -t 4
 
 And the file ioz.cfg contains these records (where /mnt/glusterfs is the
-Gluster mountpoint on each test machine):
+Gluster mountpoint on each test machine). Each record in the file is a thread in IOZone terminology. Since we have defined the number of threads to be 4 in the above example, we have four records(threads) for a single client.
 
         g01  /mnt/glusterfs  /usr/local/bin/iozone
```

Comment:
(nit) Not introduced with this change, but to keep the example and the CLI at line 170 the same, could we change g01 here to my-ip-address. What would be better is to define what g01 is, so that it is different than my-ip-address (i.e a remote machine than the one from where iozone is being run).

IOW, change this to something like,

"As an example consider a client to have the IP address test-client-ip, the ioz.cfg would hence contain,

test-client-ip /mnt/glusterfs/ ... (repeated 4 times)."

Sort of makes this better reading IMO.